src/main/java/com/example/helloworld/HelloWorldController.java

```diff
@@ -0,0 +1,64 @@
+package com.example.helloworld;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+@RestController
+public class HelloWorldController {
+
+    @GetMapping("/hello")
+    public String hello() {
+        String message = "hello world";
+        return message;
+    }
+
+    public final static LocalTime MORNING = LocalTime.of(8, 59);
+    public final static LocalTime NOON = LocalTime.of(17, 59);
+    public final static LocalTime NIGHT = LocalTime.of(23, 59);
+    public final static LocalTime MIDNIGHT = LocalTime.of(04, 00);
+
+    final DateTimeFormatter timeFormat = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
```

Comment:
こちらもprivateなstatic定数にしましょうか。
```suggestion
    private static final DateTimeFormatter TIME_FORMAT = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
```